manual/detailedTopics/build/AggregatingReverseRouters.md

```diff
@@ -1,14 +1,32 @@
 <!--- Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com> -->
+<!--
 # Aggregating reverse routers
+-->
+# リバースルートの統合
 
+<!--
 In some situations you want to share reverse routers between sub projects that are not dependent on each other.
+-->
+いくつかの状況において、互いに依存していないサブプロジェクト間でリバースルートを共有したくなることがあります。
 
+<!--
 For example, you might have a `web` sub project, and an `api` sub project.  These sub projects may have no dependence on each other, except that the `web` project wants to render links to the `api` project (for making AJAX calls), while the `api` project wants to render links to the `web` (rendering the web link for a resource in JSON).  In this situation, it would be convenient to use the reverse router, but since these projects don't depend on each other, you can't.
+-->
+例えば、`web` サブプロジェクトと `api` サブプロジェクト があるとします。`web` プロジェクトは `api` プロジェクトへのリンクを (AJAX 呼び出しのために) レンダリングする一方で、`api` プロジェクトは `web` へのリンク (JSON 内のリソースに対する web リンク) をレンダリングすることを除いては、これらのサブプロジェクトはお互いに依存関係を持ちません。このような場合、リバースルータを使うと便利ですが、これらのプロジェクトはお互いに依存していないため、使えません。
 
+<!--
 Play's routes compiler offers a feature that allows a common dependency to generate the reverse routers for projects that depend on it so that the reverse routers can be shared between those projects.  This is configured using the `aggregateReverseRoutes` sbt configuration item, like this:
+-->
+Play のルートコンパイラには、依存するプロジェクト用にリバースルートを生成する共通の依存性を提供する機能があるので、これらのプロジェクト間でリバースルートを共有することができます。以下のように、`aggregateReverseRoutes` sbt 設定項目を使って設定することができます:
```

Comment:
`Play's routes compiler`  `offers a feature`  `that allows a common dependency to generate the reverse routers` `for projects that depend on it`  `so that` `the reverse routers can be shared between those projects`

`Play のルートコンパイラは` `機能を提供します` `共通の依存性がリバースルータを生成することを許可する` `リバースルータに依存するプロジェクト用に` `このため` `リバースルータはこれらのプロジェクト間で共有されます`

内容は理解できるんだけど、うまくまとめられない^^; いい案ないでしょうかー